tests/device/test_libc/memmove1.c

```diff
@@ -0,0 +1,194 @@
+/* A minor test-program for memmove.
```

Comment:
The name of this file looks inconsisten vs. the others of its kind (memmove1.c vs. e.g.: memcpy-1.c).
Minor, but better addressed now than later :)

Reply:
I'd rather leave the name as-is, because this is the actual name of a test in newlib self-test.